app/scripts/modules/core/src/styleguide/src/styles/grids.css

```diff
@@ -0,0 +1,139 @@
+/*
+@styleguide
+
+@title Grids
+
+Offers a skeleton grid system with header, navigation and area for contents.
+Once this is set up, the contents area is also divided as a grid with 12 columns and 1 row.
+
+For setting up the main grid system for the page use the following: E.g.
+<div class="styleguide-grid-cover">
+  <main class="main-container">
+    <header>Header</header>
+    <div class="secondary-nav">Secondary Nav </div>
+    <div class="tertiary-nav">Tertiary Nav </div>
+    <section class="contents content-container with-filter with-inspector">
+      <div class="filter-area">Filters</div>
+      <div class="contents-area">Contents</div>
+      <div class="inspector-area">Inspector</div>
+    </section>
+  </main>
+</div>
+
+    <div class="styleguide-grid-cover">
+      <main class="main-container">
+        <header>Header</header>
+        <div class="secondary-nav">Secondary Nav </div>
+        <div class="tertiary-nav">Tertiary Nav </div>
+        <section class="contents content-container with-filter with-inspector">
+          <div class="filter-area">Filters</div>
+          <div class="contents-area">Contents</div>
+          <div class="inspector-area">Inspector</div>
+        </section>
+      </main>
+    </div>
+*/
+
+.main-container {
+  position: absolute;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 42px 80px 68px 1fr;
```

Comment:
I don't think we can guarantee the height of the header and nav rows, unfortunately - can these be set to `auto`, leaving the `1fr` for the `contents`?

Also worth considering where the blesk notification would go (just need to make sure it's inside the `secondary-nav`, maybe)